interface/forms/vitalsM/table.sql

```diff
@@ -21,4 +21,4 @@ CREATE TABLE IF NOT EXISTS `form_vitalsM` (
 `head_circ`         FLOAT(5,2)      default 0,
 `oxygen_saturation` FLOAT(5,2)      default 0,
 PRIMARY KEY (id)
-) ENGINE=MyISAM;
+) ENGINE=InnoDB;
```

Comment:
Also need to convert to InnoDB in the library/standard_tables_capture.inc script.